docker/docker-compose/tls/docker-compose.yml

```diff
@@ -100,7 +100,7 @@ services:
         restart: always
 
     rabbitmq:
-        image: rabbitmq:management-alpine
+        image: rabbitmq:3.8-management-alpine
```

Comment:
The changes to this file do 2 things:

* Pin the rabbitmq version to 3.8, which is the latest version that actually works with the environment variable configuration we have defined.
* Requires client certificates to be present for rabbitmq.

These changes aren't really testable yet, but it mostly doesn't matter.  Basically this is intended to be an easy way to fire up a beergarden stack to play around with.  This file won't work because it uses the latest bg docker image release, which as of right now still wouldn't actually have the rest of the changes from here or brewtils.  Once we make the next release, we can try try using this file and further tweak if necessary.  It's already broken as-is and certainly won't get more broken by what's here.